data/json/construction.json

```diff
@@ -2020,16 +2019,18 @@
                        [ { "id": "DIG", "level": 3 } ]
                        ],
         "tools" : [
-            [ "jackhammer", "jacqueshammer" ],
+            [ [ "jackhammer", 30 ], [ "jacqueshammer", 30 ], [ "elec_jackhammer", 30 ] ],
             [ "crowbar", "halligan"],
             [ "miner_hat", "hat_hard", "firehelmet", "helmet_ball",
-            "helmet_army", "helmet_lobster", "helmet_riot"]
+            "helmet_army", "helmet_lobster", "helmet_riot", "depowered_helmet", "helmet_bone",
+            "helmet_chitin", "helmet_hsurvivor", "helmet_survivor", "helmet_xlsurvivor", "miner_hat_on",
+            "power_armor_helmet_basic", "power_armor_helmet_heavy", "power_armor_helmet_light", "tac_fullhelmet",
+            "helmet_barbute", "helmet_plate", "helmet_kabuto"]
```

Comment:
I'm gonna suggest that the helmets should be open-face or powered-on.  Mining is tough work and it helps if you can breathe easily.